files/scripts/arp_update

```diff
@@ -46,5 +46,26 @@ while /bin/true; do
       ip6cmd="ip -6 neigh show | grep -v fe80 | grep $vlan | cut -d ' ' -f 1,3 | $ndisc6cmd"
       eval `eval $ip6cmd`
   done
+
+  # refresh neighbor entries from APP_DB in case of mismatch with kernel
+  DBNEIGH=$(sonic-db-cli APPL_DB keys NEIGH_TABLE*)
+  KERNEIGH4=$(ip -4 neigh show | grep Vlan | cut -d ' ' -f 1,3  --output-delimiter=',')
```

Comment:
In what case we will hit mismatch?  If some netlink messages was lost, it would be good to understand why. 

I am concerned that if we use super set of kernel and appDB for probing, we could end up with memory leak. I,E, the appDB would gone bigger and bigger,  and nobody would ever remove the entries.

Reply:
I don't think so. when we ping for an entry that is not present in kernel but in app_db and the ping/arp fails, it will be removed from app_db. So next iteration will not have this entry in app_db. 
I agree we have to root cause the losing of netlink message. We will try to simulate some conditions with heavy cpu load but this is kind of a WA

Reply:
While we are root causing the lose of netlink messages. I am  trying to understand the impact of this scenario.  If the neighbor entry is not in kernel but in app-DB, what was exactly the issue for traffic?   Is it because the traffic will be send to a wrong mac instead of trapping to CPU for arp, so we want to remove the neighbor in HW?  
Anyway, the ping is way too slow here unless we do parallel ping.    How many neighbor entries on DUT where we hit this issue?

Reply:
Yes, it can either be send to wrong mac or mac may age out and hw points to drop action. In our observation, it is around ~100 entries. So with 0.2 sec timeout, we are talking about 20 sec which is within the 300 sec sleep. I don't think we need to have a parallel ping here. I also thought of changing timeout to 0.1 sec but doesn't seem to be really necessary as this is kind of a safety mechanism and not something that keeps happening in production.

Reply:
any concern of 0.1 second?